pyperformance/cli.py

```diff
@@ -167,13 +167,13 @@ def parse_args():
     if hasattr(options, 'python'):
         # Replace "~" with the user home directory
         options.python = os.path.expanduser(options.python)
-        # Try to the absolute path to the binary
-        abs_python = shutil.which(options.python)
+        # Try to get the absolute path to the binary
+        abs_python = os.path.abspath(options.python)
```

Comment:
Yeah, I recall that I also had issues when I resolved symlinks of env/bin:python. I don't recall when this binary is a symlink or not. I just recall that it should not always resolve symlinks ;-)